config.yaml

```diff
@@ -28,9 +28,9 @@ platform:
 # Modules for loading model executables
 modules:
   use:
-      - /g/data/vk83/modules
+      - /g/data/vk83/prerelease/modules
```

Comment:
will replace with build of new main: However this was the testing build used in PR of latest merge to main